compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1794,6 +1823,7 @@ crate fn show_candidates(
     candidates: &[ImportSuggestion],
     instead: bool,
     found_use: bool,
+    is_pattern: bool,
```

Comment:
nit: it might be a bit more readable if these `bool` parameters were replaced with some simple enums